test/DynamicDependency/Test_Win32/Test_GetCurrentPackageInfo.cpp

```diff
@@ -343,14 +352,27 @@ namespace Test::DynamicDependency
             _Out_opt_ UINT32* count);
         GetCurrentPackageInfo3Function m_getCurrentPackageInfo3{};
 
-        void EnsureGetCurrentPackageInfo3IsIniitalizedIfExists()
+        typedef UINT32 (WINAPI* GetPackageGraphRevisionIdFunction)();
+        GetPackageGraphRevisionIdFunction m_getPackageGraphRevisionId{};
+
+        void EnsureKernelbaseDllIsLoaded()
         {
             if (!m_kernelbaseDll)
             {
                 wil::unique_hmodule dll{ LoadLibraryW(L"kernelbase.dll") };
                 FAIL_FAST_HR_IF_NULL(E_UNEXPECTED, dll);
```

Comment:
Don't you want to log GetLastError() instead for debugging purposes instead of bucketing them to E_UNEXPECTED ?, when Kernel base dll couldn't be found ?

Reply:
Hmmm. Technically yes, though it's kernelbase.dll which is loaded into a win32 process before even `DllMain()` is invoked so this should just hit the Loaded Module Table and find the match. Or rather, if it doesn't Windows is in a seriously broken state already and I'd expect maybe and bloodshed well before reaching this point ;-)